lib/transformers/transformers.js

```diff
@@ -145,16 +120,48 @@ const mapOutRes = ({ ctx, health, bodyKeys, headersRegex }) => {
     });
   }
 
+  const meta = extractRequestMeta(ctx, bodyKeys, headersRegex, 'request');
+
   let contentLength = ctx.state.calculatedContentLength || 0;
   let userAgent = getUserAgent(ctx.headers);
 
-  return Object.assign({ status, duration, headers }, ctx.logCtx, {
+  return Object.assign({ status, duration, headers }, ctx.logCtx, meta, {
```

Comment:
🙃  If `inbound.request.enabled` is true do we need headers and bodykeys here?

Reply:
it will be redundant, but it will be consisting. If someone enables inbound request, he will miss logs from the outbound response.